app/components/no-mobile-message/template.hbs

```diff
@@ -0,0 +1,12 @@
+<div>
+  {{fa-icon 'mobile'}}
+  <h3>Sorry, this page doesn't work on mobile!</h3>
+  <p>
+    The data on this page is taking up too much space for your currently used
+    device.
+  </p>
+  <p>
+    Try opening this page on a larger device. or take me back to
```

Comment:
I'd suggest "Try opening this page on a larger device, or go back to the home page." - you're refering to the user as "you" in the sentence above.